packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -536,15 +571,31 @@ function TokensPanel({
   }, [tokensFromLists, tokensFromUser, newToken, getBalance])
 
   const storeNewToken = async () => {
-    return token.add(newToken).catch((ex: Error) => {
-      let error = 'Token not found on this network.'
+    let error = 'Token not found on this network.'
+    let didSucceed = false
 
+    try {
+      // Try to add the token as an L2-native token
+      token.addL2NativeToken(newToken)
+      didSucceed = true
+    } catch (error) {
+      //
```

Comment:
remove empty comment?

Reply:
I'm curious about this too but I think it forces `.find` to go around linter and stay in the line below (more readable) ?

Reply:
It looks weird without haha